examples/rate-limit/main.go

```diff
@@ -76,8 +81,10 @@ func main() {
 			Steps: []*worker.WorkflowStep{
 				worker.Fn(StepOne).SetName("step-one").SetRateLimit(
 					worker.RateLimit{
-						Units: 1,
-						Key:   "api1",
+						Key:            "per-user-rate-limit",
+						KeyExpr:        &keyExpr,
```

Comment:
accepts both a key and a key expr at the same time?

Reply:
Resolved offline